guardian/support-service-lambdas
# Create Lambda for processing failed national deliveries

## What does this change?
Add a typescript skeleton lambda. Basic Hello World output is present in lambda to verify that it can produce output.

## Background
- part of the National Delivery project to begin delivering papers outside the M25
- first stage of building the Failed Deliveries Processor for handling failed deliveries

## How to test
Navigate to lambda, hit 'Test' button and verify that 'Hello world' is output to screen and no errors occur

## How can we measure success?
Lambda successfully runs when 'Test' button is pressed and 'Hello world' is output to screen and no errors occur

## Coming next
- retrieve a file from S3
